pkgs/top-level/python-packages.nix

```diff
@@ -23341,7 +23341,7 @@ in {
     };
   };
 
-  Theano = buildPythonPackage rec {
+  theano = buildPythonPackage rec {
```

Comment:
Changing bound names like this can be annoying, consider either not doing it or adding an alias.